src/random.cpp

```diff
@@ -10,7 +10,7 @@
 #include <crypto/sha512.h>
 #include <support/cleanse.h>
 #ifdef WIN32
-#include <compat.h> // for Windows API
+#include <compat/compat.h> // for Windows API
```

Comment:
Maybe the comment `// for Windows API` is too narrow and should be removed? (here and elsewhere)

Reply:
Agree. If it was specifically about windows compatibility, we could name it like `wincompat.h` instead of having comments like this. But I think the file is less specific?

Reply:
Will drop this comment.